3-projects/cloudbuild-tf-apply.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 1200s
+steps:
+- name: gcr.io/$PROJECT_ID/terraform
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - |
+    gcloud config set auth/impersonate_service_account ${_TF_SA_EMAIL}
```

Comment:
This is nice and more concise, although you can no longer select a specific step in the Cloud Build UI with each step grouped like this